vignettes/articles/presentations/technical-demo/technical-demo.qmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "Technical Demo"
+subtitle: "Contraceptives DDI Trial Simulation Platform"
+date: 2024-08-27
+date-modified: today
+author: Felix MIL
+institute: |
+  ![](https://esqlabs.com/wp-content/uploads/2018/05/ESQLABS_final_600px.png){height=100}
+embed-resources: true
+format: esqlabs-revealjs
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message = FALSE)
+library(cts)
+```
+
+# 🚩 Introduction
+
+## 👥 Presentations
+
+::: {layout-ncol="3"}
+![Pavel B. - Lead Scientist](img/Pavel.png){style="border-radius: 50%;" width="40%"}
+
+![Felix M. - Project Lead/R Dev.](img/Felix.png){style="border-radius: 50%;" width="40%"}
+
+![Diane L. - Scientist/R Dev](img/Diane.png){style="border-radius: 50%;" width="40%"}
+
+![Rob M. - C# Dev](img/Rob.png){style="border-radius: 50%;" width="40%"}
+
+![Rudolf E. - R Dev](img/Rudolf.png){style="border-radius: 50%;" width="40%"}
+:::
+
+## 🌐 Project Overview
+
+### 🎯 Goal
+
+Develop a clinical trial simulator (**CTS**) for drug-drug interactions (DDI) for contraceptives using physiologically based pharmacokinetic (PBPK) models snapshots.
+
+### 📍 Scope
+
+-   R package
+-   Programming Interface
+
+### 📅 Deadline
+
+31th October 2024
+
+------------------------------------------------------------------------
+
+### 📦 Deliverables
+
+-   R package
+-   User documentation
+-   Test suite
+-   Continuous integration workflows (GitHub Actions)
+-   Git repository with entire history
+
+# Concept {.smaller}
+
+------------------------------------------------------------------------
+
+### 📥 Inputs
+
+Compound models snapshots (`.json` files) from:
+
+-   OSP model library (hosted on GitHub)
+-   Local file or url
+
+### 📤 Outputs
+
+-   DDI model snapshot (`.json`) or project (`.pksim5`)
+-   Simulation results (`.csv`)
+-   Simulation file (`.pkml`)
+-   PK statistics summary
+-   Figures
+
+------------------------------------------------------------------------
+
+### 🗺️ Concept Map
+
+![](../../img/CTS-concept-map.png){fig-align="center"}
+
+# ✅Requirements {.smaller}
+
+## 👥 User Requirements
+
+-   Easily install the package,
+-   Browse and select compounds from the OSP model library,
+-   Import compounds snapshots from a json file path or a url,
+-   Review the compound model parameters,
+-   Review the DDI model parameters,
+-   Parameterize some aspects of the DDI simulation
+-   Export the DDI model as a Project Snapshot,
+-   Import a DDI model snapshot,
```

Comment:
You mean from a DDI repository on GitHub, like this? https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Quinidine-DDI

No this is not required.

Reply:
No, I mean reimporting a DDI model snapshot that was created with this tool before. For example, to change study design or other parameter before re-exporting it.